docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_creating_custom_test.adoc

```diff
@@ -91,6 +91,18 @@ spec:
 . Add your new custom test as an integration test in {ProductName}.
 .. For additional instructions on adding an integration test, see this document: xref:how-to-guides/testing_applications/proc_adding_an_integration_test.adoc[Adding an integration test].
 
+----
+In the integration test created above, integration-service adds some Params/Labels automatically to the integration PipelineRun
+
+1. The param of "SNAPSHOT" containing the Snapshot as a json string
+2. The param of "NAMESPACE" containing the namespace where the snapshot will be deployed 
+3. The Label of "appstudio.openshift.io/snapshot" containing the name of snapshot
+4. The Label of "test.appstudio.openshift.io/optional" for optional flag of IntegrationTestScenario
+5. The Label of "test.appstudio.openshift.io/scenario" containing the name of IntegrationTestScenario
+6. The Label of "appstudio.openshift.io/application" containing the name of application
+7. The Label of "appstudio.openshift.io/component" containing the name of component
```

Comment:
```suggestion
.Data injected into integration PipelineRun

In the integration test created above, integration-service adds following items automatically to the integration PipelineRun.

Parameters:

. `SNAPSHOT` containing the Snapshot as a json string
. `NAMESPACE` containing the namespace where the snapshot will be deployed 

Labels:
. `appstudio.openshift.io/application` containing the name of application
. `appstudio.openshift.io/component` containing the name of component
. `appstudio.openshift.io/snapshot` containing the name of snapshot
. `test.appstudio.openshift.io/optional` for optional flag of IntegrationTestScenario
. `test.appstudio.openshift.io/scenario` containing the name of IntegrationTestScenario
```

Reply:
The format is not working well with the suggested change.